source/reference/minio-mc/mc-stat.rst

```diff
@@ -91,17 +92,18 @@ Parameters
 
       mc stat ~/data/myobject.txt
 
-.. mc-cmd:: --encrypt-key
+.. mc-cmd:: --enc-c
    :optional:
 
-   Encrypt or decrypt objects using server-side encryption with client-specified keys. Specify key-value pairs as ``KEY=VALUE``.
-   
-   - Each ``KEY`` represents a bucket or object. 
-   - Each ``VALUE`` represents the data key to use for encrypting object(s).
+   Encrypt or decrypt objects using client provided keys.
+   Repeat the flag to pass multiple keys.
+
+   Keys must be in either Raw Base64 or Hex format.
 
-   Enclose the entire list of key-value pairs passed to :mc-cmd:`~mc stat --encrypt-key` in double quotes ``"``.
+.. mc-cmd:: --no-list
+   :optional:
 
-   :mc-cmd:`~mc stat --encrypt-key` can use the ``MC_ENCRYPT_KEY`` environment variable for retrieving a list of encryption key-value pairs as an alternative to specifying them on the command line.
+   Disable all ``LIST`` operations if an object does not exist.
```

Comment:
Perhaps something like this? 🤔  You can do stat against a bucket, prefix, or object.
```suggestion
   Disable all ``LIST`` operations if the target does not exist.
```

From the issue that prompted this feature request, I believe they were trying to stat a ton of things by name. And many of them didn't exist. The unneeded `list()` was a performance concern.

Reply:
Drive-by fix on line 83, which I can't comment on:

> bucket or bucket prefixy

Reply:
Bucket prefixy. 😆